internal/export/distro/registrations.go

```diff
@@ -249,17 +262,11 @@ func Loop(errChan chan error, eventCh chan *models.Event) {
 
 	registrations := make(map[string]*registrationInfo)
 
-	allRegs := getRegistrations()
+	allRegs, err := getRegistrations()
 
-	for allRegs == nil {
-		logger.Info("Waiting for client microservice")
-		select {
-		case e := <-errChan:
-			logger.Info("exit msg", zap.Error(e))
-			return
-		case <-time.After(time.Second):
-		}
-		allRegs = getRegistrations()
+	if err != nil {
```

Comment:
export distro needs export client to be running to start. It is not perfect, but until we decide how are we going to solve this problem across all the microservices this should not be removed.